flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliStrings.java

```diff
@@ -52,7 +52,7 @@ private CliStrings() {
 		.append(formatCommand(SqlCommand.RESET, "Resets all session configuration properties."))
 		.append(formatCommand(SqlCommand.SELECT, "Executes a SQL SELECT query on the Flink cluster."))
 		.append(formatCommand(SqlCommand.SET, "Sets a session configuration property. Syntax: 'SET <key>=<value>;'. Use 'SET;' for listing all properties."))
-		.append(formatCommand(SqlCommand.SHOW_FUNCTIONS, "Shows all registered user-defined functions."))
+		.append(formatCommand(SqlCommand.SHOW_FUNCTIONS, "Shows all system-defined and registered user-defined functions."))
```

Comment:
How about "Show all user-defined and built-in functions". 

Usually, we call it "built-in" instead of "system-defined".

Reply:
Thanks, @wuchong, sounds good to me, done by  [868ce4](https://github.com/apache/flink/pull/9752/commits/868ce4a4f6149db263a8107fa3a1de3c079d7916)

Reply:
@wuchong fyi: actually we just had a discussion on the ML around temporary functions where we decided to call the built-in functions "system" functions. Like `CREATE TEMPORARY SYSTEM FUNCTION` if users want to override a built-in function.